components/automate-gateway/api/iam/v2/response/rules.proto

```diff
@@ -34,6 +44,9 @@ message ApplyRulesStartResp {}
 message ApplyRulesCancelResp {}
 
 message ApplyRulesStatusResp {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    example: { value: '{ "state": "running", "estimated_time_complete": "2020-03-20T19:24:55Z", "percentage_complete": 0.5, "failed": false, "failure_message": "", "cancelled": false}'}
+  };
   string state = 1;
   google.protobuf.Timestamp estimated_time_complete = 2;
   float percentage_complete = 3;
```

Comment:
The fields of this particular response also need to be annotated because they are simple types.
<img width="435" alt="image" src="https://user-images.githubusercontent.com/6817500/77229581-114a7400-6b4c-11ea-9a47-eece3fbf6509.png">

Per this note:
https://github.com/chef/automate/blob/e79891a6b5e7de39488e322f5a7e509b46d10e2d/components/automate-gateway/api/iam/v2/response/policy.proto#L11
you will see in that policy.proto, for example, that only the simple types there are annotated. (For further details there are a couple paragraphs discussing this in #2982.)